stacks/serverless/root.yml

```diff
@@ -134,3 +134,35 @@ Resources:
           Value: !Ref AWS::StackId
       TemplateURL: !Join ["", [!Ref TemplateUrlPrefix, "dovetail-counts-lambda.yml"]]
       TimeoutInMinutes: 5
+  CmsAudioLambdaStack:
+    Type: "AWS::CloudFormation::Stack"
+    Properties:
+      NotificationARNs:
+        - Fn::ImportValue:
+            !Sub "${InfrastructureNotificationsStackName}-CloudFormationNotificationSnsTopic"
+      Parameters:
+        AlarmWarnActions:
```

Comment:
This passes in a String as the parameter ... but the `CommaDelimitedList` should transform it into a 1-item list.